### DISCUSSION TITLE
Rednotebook Poor Display

### DISCUSSION THREAD

**Discussion by lesdelc** [jendrikseipp/rednotebook#764] [General]
I downloaded to Archlinux by Flatpak the latest version of Rednotebook v2.33. Please view the downloaded attachment when I load and display Rednotebook it is not clearly displayed. The font Rednotebook is using is a poor font. When I switch from Wayland to  X.org the display is crisp and clear. See attachment. How can I resolve the problem when using Wayland?

![RednoteBook](https://github.com/jendrikseipp/rednotebook/assets/59441829/7cdb5836-8826-48ea-a86b-b61ccfd4223d)

-Les-

**Comment by jendrikseipp** [comment]
If you start the app via the command line, are there any warnings or errors?

**Reply by lesdelc** [reply]
How do I start the app from the command line?

**Reply by jendrikseipp** [reply]
`flatpak run app.rednotebook.RedNotebook`

**Reply by lesdelc** [reply]
I just started to use Flatpak. Thanks for the help. Tried it no change in the display.

Is this the problem?
(rednotebook:2): Gtk-CRITICAL **: 09:52:40.413: gtk_widget_get_scale_factor: assertion 'GTK_IS_WIDGET (widget)' failed

Les
